apollo-router/src/plugins/telemetry/metrics/apollo/studio.rs

```diff
@@ -73,7 +73,7 @@ pub(crate) struct SingleQueryLatencyStats {
     pub(crate) private_cache_ttl_latency: Option<Duration>,
     pub(crate) registered_operation: bool,
     pub(crate) forbidden_operation: bool,
-    pub(crate) without_field_instrumentation: bool,
+    pub(crate) requests_without_field_instrumentation: u64,
```

Comment:
I'm not sure I follow this. The other fields on this struct (and its title) seem like they are describing a single request, not multiple requests.  eg, how can `cache_hit` still be a bool if this now is a count of multiple requests?

Reply:
I’ve been wondering the same.

This struct describes a single *supergraph* request and response, but that may involve multiple subgraph requests each returning (or not) an FTV1 `Trace`. Here I tried to port https://github.com/apollographql/apollo-server/blob/fa82c1d5299/packages/server/src/plugin/usageReporting/stats.ts#L212-L214 by counting how many subgraph requests returned a trace with `fieldExecutionWeight == 0`. Should this be counting supergraph requests instead? If not, should it also count subgraph requests that do not return an FTV1 trace at all?

Reply:
Very little in the usage reporting protocol is federation-aware. In Apollo Server this is counting overall end-user (supergraph) requests, not subgraph requests, and this should probably do the same. Note that in AS this is counting "we decided to not ask for ftv1 on this particular operation", not "we actually received ftv1 back from subgraphs"... probably reasonable to continue that choice here.

Reply:
I’ve reverted the `without_field_instrumentation` change in order to get the rest finished up, I’ll make a follow-up PR